apps/nextjs-app/src/features/app/blocks/view/grid/components/RecordMenu.tsx

```diff
@@ -89,6 +90,18 @@ export const RecordMenu = () => {
           insertRecord?.(record.id, 'after');
         },
       },
+      {
+        type: MenuItemType.Duplicate,
+        name: t('table:menu.duplicate'),
+        icon: <Copy className={iconClassName} />,
+        hidden: isMultipleSelected || !permission['record|create'],
+        disabled: isAutoSort,
```

Comment:
If any field contains a unique constraint and the field in the copied record has values, the copying action cannot be performed.

Reply:
Please tell me, what other permissions should be added